### ISSUE TITLE
[CLIFF]I would like to ask you a question about the Agora dataset

### ISSUE THREAD

**Issue by cyk990422** [huawei-noah/noah-research#157]
Hello, Dr. Lee！
I would like to ask you whether it is necessary to perform finetuning on the yolov3 detector when performing finetuning on the Agora dataset.

**Comment by zhihaolee** [comment]
We describe it in the supplementary material, "As demonstrated in the AGORA evaluation, we use the BBox predicted by Mask R-CNN which is trained on COCO without finetuning on AGORA".

**Comment by cyk990422** [comment]
Thank you very much for your reply!
Dr. Lee, I have another question.
When you are fine-tuning Agora, how do you deal with the kid data in it? I think their beta parameters are 11-dimensional. How do you deal with it?

**Comment by zhihaolee** [comment]
We just treat the kids as adults.

**Comment by cyk990422** [comment]
Thank you for such a quick reply! Happy new year!

**Comment by cyk990422** [comment]
Sorry, Dr. Lee.
May I ask, when you processed the kids data into adults, did you directly take the first ten digits of the betas parameter, or did you use another method?

**Comment by cyk990422** [comment]
Sorry, Dr. Lee.
May I ask, when you processed the kids data into adults, did you directly take the first ten beta parameters, or did you use another method?

**Comment by zhihaolee** [comment]
We don't distinguish kids from adults, in other words, we don't know whether the detected person is a kid or an adult, so we use ten beta parameters for all persons.

**Comment by yufu-wang** [comment]
The training data of Agora also includes kid samples. During finetuning on Agora,
1. Did you include those kid samples for training?
2. If so, did you only use the first ten beta parameters to generate 3D/2D joints for training? Or did you use the kid smpl model to generate 3D/2D joints for training?
Thank you!
